qCC/mainwindow.cpp

```diff
@@ -8934,10 +8940,22 @@ void MainWindow::doActionCloudPrimitiveDist()
 				if (!(returnCode = CCLib::DistanceComputationTools::computeCloud2SphereEquation(compEnt, refEntity->getOwnBB().getCenter(), static_cast<ccSphere*>(refEntity)->getRadius(), signedDist)))
 					ccConsole::Error(errString, "Sphere", returnCode);
 				break;
-			case CC_TYPES::PLANE:
-				if (!(returnCode = CCLib::DistanceComputationTools::computeCloud2PlaneEquation(compEnt, static_cast<ccPlane*>(refEntity)->getEquation(), signedDist)))
-					ccConsole::Error(errString, "Plane", returnCode);
+			case CC_TYPES::PLANE: 
+			{
+				ccPlane* plane = static_cast<ccPlane*>(refEntity);
+				if (treatPlanesAsBounded)
+				{
+					CCLib::SquareMatrix rotationTransform(plane->getTransformation().data(), true);
+					if (!(returnCode = CCLib::DistanceComputationTools::computeCloud2RectangleEquation(compEnt, plane->getXWidth(), plane->getYWidth(), rotationTransform, plane->getCenter(), signedDist)))
+						ccConsole::Error(errString, "Bounded Plane", returnCode);
```

Comment:
Style: Missing "{}" (throughout the switch statement)

Also shouldn't perform assignment inside an if statement - though I see that this is copying the existing style from the surrounding code. This switch statement should be cleaned up (in a separate PR).